bigquery/tests/unit/test_magics.py

```diff
@@ -257,6 +281,48 @@ def test_bigquery_magic_clears_display_in_verbose_mode():
         assert clear_mock.call_count == 1
 
 
+@pytest.mark.usefixtures("ipython_interactive")
+@pytest.mark.skipIf(
+    bigquery_storage_v1beta1 is None, "Requires `google-cloud-bigquery-storage`"
+)
+def test_bigquery_magic_with_bqstorage(monkeypatch):
```

Comment:
Can you add a test or update an existing test asserting that the BQ storage API is not used when the new parameter is not set?